src/pages/home/sidebar/SidebarScreen/FloatingActionButtonAndPopover.js

```diff
@@ -56,51 +57,30 @@ const propTypes = {
 
     /** Indicated whether the report data is loading */
     isLoading: PropTypes.bool,
+
+    /** FAB ref for access from HOC */
```

Comment:
```suggestion
    /** Forwarded ref to FloatingActionButtonAndPopover  */
```

HOC would not be the right word here @dayana7204

Reply:
Or `Ref to FloatingActionButtonAndPopover`